

<img src="./logo_UTN.svg" align="right" width="150" /> 
 Medidas Electrónica I

# Ejercicios de métodos de medición
#### Israel Pavelek 

Prof: Ing. Perdomo Juan

Ayudante: Ing. Franco Zaccra


### Ejercicio 1

Se midió una resistencia con un voltimetro y  una fuente de corriente de 1A ±0,1% y RS = 1MΩ, obteniéndose los siguientes datos: 

VOLTÍMETRO

<table>
  <tr>
    <td>Dígitos</td>
    <td>4,1/2</td>
  </tr>
  <tr>
    <td>Rangos</td>
    <td>50mV; 500mV, 5V, 50V, 500V y 1000V</td>
  </tr>
  <tr>
    <td>Error</td>
    <td>±(0,025% + 2d)</td>
  </tr>
  <tr>
    <td>Ri</td>
    <td>10MΩ</td>
  </tr>
  <tr>
    <td>Indicaciones</td>
    <td>48,329mV - 48,327mV - 48,346mV - 48,318mV - 48,323mV - 48,321mV</td>
  </tr>
</table>

Se solicita:
* 1_Determine el valor de la resistencia y su incertidumbre.
* 2_Determine el error de método cometido e indique si es necesario corregirlo




In [2]:
import numpy as np
from sympy import Derivative, diff, simplify, symbols,S
import statistics

Im=1
V=[48.329,48.327,48.346,48.318,48.323,48.321]
N=len(V)
Vm=np.mean(V)
Rm=Vm/(Im)
SV=statistics.stdev(V)
uaV=SV/np.sqrt(N)
print('Vm=',Vm,'mV') 
print('Rm=',Rm,'mΩ') 
print('S(V)=',SV,'mV') 
print('ua(V)=',uaV,'mV') 

Vm= 48.327333333333335 mV
Rm= 48.327333333333335 mΩ
S(V)= 0.009973297682645094 mV
ua(V)= 0.004071581729227066 mV


### TIPO B

In [3]:
DELTAV=((0.025/100)+(2/48327))*Vm
ubV=(DELTAV/np.sqrt(3))
DELTAI=(0.1/100)*Im
ubI=(DELTAI/np.sqrt(3))
print('ub(V)=',ubV,'mV') 
print('ub(I)=',ubI,'A')


ub(V)= 0.008130158230179058 mV
ub(I)= 0.0005773502691896258 A


### Combinando

In [4]:
ucV=np.sqrt(ubV**2+uaV**2)
ucVr=ucV/Vm
ucI=ubI
ucIr=ucI/Im
ucRr=np.sqrt(ucVr**2+ucIr**2)
ucR=ucRr*Rm
print('ucr(V)=',ucVr)
print('ucr(I)=',ucIr)
print('ucr(R)=',ucRr)
print('uc(R)=',ucR,'mΩ')




ucr(V)= 0.00018814824915159458
ucr(I)= 0.0005773502691896258
ucr(R)= 0.0006072339721986443
uc(R)= 0.02934599858576795 mΩ


In [5]:
K=2
U=ucR*K
print("U=",U,'mΩ')

U= 0.0586919971715359 mΩ


In [6]:
print(f"R=({Rm:.3f}±{U.round(3):.3f})mΩ")
print("K=2 con el 95,45% de nivel de confianza")

R=(48.327±0.059)mΩ
K=2 con el 95,45% de nivel de confianza


Para el punto 2, veamos el circuito de medición:

<img src="./Ej1em.png" align="center" width="500" /> 

Donde V es la tensión que efectivamente mediría el voltímetro.

En este caso la corriente que entregaría la fuente de 1A sería repartida, en principio por RS y RV ademas de la R que estamos queriendo medir.

$$em_r=\frac{Ri-Rv}{Rv}$$

$$em_r=\frac{Ri}{Rv}-1$$

$Ri=\frac{Vi}{Ig}$

Llamamos Rp al paralelo de Rv y Rs

$em_r=\frac{\frac{Vi}{Ig}}{\frac{Vi}{\frac{Rp}{Rp+Ri}\cdot Ig}}-1$

$em_r=\frac{1}{\frac{Rp+Ri}{Rp}}-1$

$em_r=\frac{Rp}{Rp+Ri}-1$

$em_r=\frac{Rp-Rp-Ri}{Rp+Ri}$

$$em_r=\frac{-Ri}{Rp+Ri}$$

$$em_r=\frac{-\frac{Vi}{Ig}}{Rp+\frac{Vi}{Ig}}$$

$$em_r=\frac{-\frac{Vi}{Ig}}{Rp+\frac{Vi}{Ig}}$$

Reemplazando:

Rp=1MΩ//10MΩ=909.09KΩ

$$em_r=\frac{-48.327m\Omega}{909.09K\Omega+48.327m\Omega}=53.16n$$


Dado que $10\cdot em_r<U(R)$

No es necesario corregir.

### Ejercicio 2

Se desea medir la tensión de un generador de señal senoidal con un voltímetro

VOLTÍMETRO:	

<table>
  <tr>
    <td>Tipo</td>
    <td>True RMS - 4½ Dígitos</td>
  </tr>
  <tr>
    <td>Rangos</td>
    <td>200mV – 2V – 20V– 200V– 1000V</td>
  </tr>
  <tr>
    <td>Error</td>
    <td>±(0,05% + 2d)</td>
  </tr>
  <tr>
    <td>Zv</td>
    <td>1MΩ // 470pF</td>
  </tr>
</table>

Generador:

<table>
  <tr>
    <td>Rg</td>
    <td>600Ω</td>
  </tr>
  <tr>
    <td>F</td>
    <td>15kHz</td>
  </tr>
</table>

Se solicita:
* 1_ Deduzca la ecuación del error de método, en función de los componentes del circuito
* 2_ Si Vi = 1,8562V, determine el error de método, ¿es necesario corregirlo? ¿cuál es el valor corregido?



$e_{Mr}=\frac{V_i-V_v}{V_v}=\frac{|V_{Zv}|-|V_{TH}|}{|V_{TH}|}=\frac{|V_{Zv}|}{|V_{TH}|}-1$

$Z_v=\frac{R_v}{j\omega R_v C_v +1}$

$V_{ZV}=\frac{Z_V}{R_G+Z_V} \cdot V_{TH}$

$\frac{|V_{Zv}|}{|V_{TH}|}=\frac{R_G}{R_G+R_V} \cdot \frac{1}{\sqrt{1+(\omega C_V (R_G // R_V))^2}}$

$e_{Mr}=\frac{R_G}{R_G+R_V} \cdot \frac{1}{\sqrt{1+(\omega C_V (R_G // R_V))^2}}-1$

$\Delta V_r=\frac{ \Delta V}{V_I}=(\frac{0,05 \%}{100}+\frac{2}{168562})=0,5118m$

$\mu_A(V_I)=0$

$\mu_{Br}(V_I)=\frac{\Delta V_r}{\sqrt3}=0,2955m$

$\mu(V_I)=\sqrt{\mu^2_B(V_I)}=\mu_B(V_I)$

$\mu_r(V_I)=\mu_{Br}(V_I)$

$U_r(V_I)=K\cdot\mu_r(V_I)=2 \cdot 0,2955m=0,5910m$

$e_{Mr}=\frac{R_G}{R_G+R_V} \cdot \frac{1}{\sqrt{1+(2\pi F C_V (R_G // R_V))^2}}-1$

$e_{Mr}=0,99940 \cdot \frac{1}{\sqrt{1+(94247,77 470pF 599,64\Omega)^2}}-1$

$e_{Mr}=-0,952369m$

Como $\frac{U_R}{10}=-0,0952369m<|e_{Mr}$

Necesito corregir

$V_V=\frac{V_I}{1+e_{Mr}}=\frac{1,8562V}{1-0,0952369m}=1,856376V$

$V_V=1,856376V$







### Ejercicio 3

Se desea determinar la regulación de línea de una fuente switching estimada en 0,25%. Tensión de entrada: 9V a 30V; Tensión de salida: 3,3V.

* a_Dibuje el circuito correspondiente y describa el proceso de medición
* b_Determinar todos los elementos necesarios para realizar el ensayo, justificando la elección.
* c_Hallar el resultado de la medición. ¿El regulador se encuentra dentro de la especificación del fabricante?

Voltímetro

<table>
  <tr>
    <td>Tipo</td>
    <td>4½ Dígitos</td>
  </tr>
  <tr>
    <td>Rangos</td>
    <td>200mV – 2V – 20V</td>
  </tr>
  <tr>
    <td>Error</td>
    <td>±(0,05% + 2d)</td>
  </tr>
  <tr>
    <td>Rv</td>
    <td>10MΩ</td>
  </tr>
</table>

Mediciones:

<table>
  <tr>
    <td>$V_{1MEDIA}=0,01mV$</td>
    <td>$s(V1)=0,06mV$</td>
    <td>$N=9$</td>
  </tr>
  <tr>
    <td>$V_{2MEDIA}=8,52mV$</td>
    <td>$s(V2)=0,09mV$</td>
    <td>$N=9$</td>
  </tr>
</table>


#### Punto A

Circuito:

<img src="./Ej3.png" align="center" width="500" /> 

Descripción del proceso de medición:

* Con VL en tensión de trabajo nominal, ajustar VREF (mediante R4 y R2) hasta lograr que la tensión indicada sea cero (en la escala de 200mv).
* Llevar VL a la tensión máxima de regulación de línea soportada y tomar la medición.
* Llevar VL a la tensión de trabajo nominal y verificar que la indicación vuelva a cero, de lo contrario se descarta la medición

#### Punto B

Elementos:
    Vs:3.3v
    VR: Utilizamos una bateria alcalina de 9V dado que posee gran establidad
    
    Dado el multimetro de 4½ digitos en la escala de 200mv tenemos una resolución de 0.01mV 
    
VR:9V

Adopto para que no sea muy grande y descargue la bateria ni pequeña para que el ruido afecte. 

$Ir=100\mu A$


$V_{R2}=2\cdot \Delta V_{e}$

$\Delta V_{e}=\frac{3,3V \cdot 0,25\%}{100}=8,25mV$

Adopto:

$V_{R2}=16,5mV$

$R_2=\frac{V_{R2}}{I_R}=\frac{16,5mV}{100\mu A}=165\Omega=150\Omega$

Con $150\Omega V_{R2}=15,0mv$

$\Delta V_{R2}=\frac{V_{R2}}{P_{R2}}=\frac{15,0mV}{100}=0,150mv$

$V_{R1}=9V-(3,3V+7,5mv)=5,6925V$

$R_1=\frac{V_{R1}}{I_R}=\frac{5,6925V}{100\mu A}=56K\Omega$


Establezco el valor de $R_4$ que me permite tener la resolución buscada:

$0,005mV=\Delta V_{R4}=\frac{V_{R4}}{P_{R4}}=\frac{V_{R4}}{100}$

$V_{R4}=0,005mV \cdot 100=0,5mV$

$R_4=\frac{V_{R4}}{I_R}=5\Omega$    

$V_{R_3}=9V-(V_{R1}+V_{R2}+V_{R4})=3,2425V$

$R_3=\frac{V_{R3}}{I_R}=32,425K\Omega=33K\Omega$ 

#### Punto C       

### TIPO A

In [7]:
V1m=0.01
SV1=0.06
uaV1=SV1/np.sqrt(N)
print('ua(V)=',uaV1,'mV') 


V2m=8.52
SV2=0.09
uaV2=SV2/np.sqrt(N)
print('ua(V)=',uaV2,'mV') 


ua(V)= 0.024494897427831782 mV
ua(V)= 0.03674234614174767 mV


### TIPO B

In [8]:
DELTAV1=((0.05/100)+(2/1))*V1m
ubV1=(DELTAV1/np.sqrt(3))

DELTAV2=((0.05/100)+(2/852))*V2m
ubV2=(DELTAV2/np.sqrt(3))
print('ub(V1)=',ubV1,'mV') 
print('ub(V2)=',ubV2,'mV')

ub(V1)= 0.011549892135138465 mV
ub(V2)= 0.014006517530540321 mV


### COMBIANANDO

In [9]:
ucV1=np.sqrt(ubV1**2+uaV1**2)
ucV2=np.sqrt(ubV2**2+uaV2**2)

print('uc(V1)=',ucV1,'mV')
print('uc(V2)=',ucV2,'mV')


uc(V1)= 0.0270813590562463 mV
uc(V2)= 0.0393215276068127 mV


In [10]:
DeltaV=V2m-V1m
uDeltaV=np.sqrt(ucV1**2+ucV2**2)
UDeltaV=2*uDeltaV
print(f"DeltaV=({DeltaV:.3f}±{UDeltaV.round(3):.3f})mV")
print("K=2 con el 95,45% de nivel de confianza")


DeltaV=(8.510±0.095)mV
K=2 con el 95,45% de nivel de confianza


El fabricante dice que:

$\Delta V_{e}=\frac{3,3V \cdot 0,25\%}{100}=8,25mV$

$\Delta V_{min}=(8,51-0,095)mV=8,415mV$

$\Delta V_{max}=(8,51+0,095)mV=8,605mV$

El dipositivo no cumple con la especificacion

# Ejercicio 4

Se desea determinar la regulación de carga de una fuente estimada en 50mV con una IMAX = 1.5A, cuya tensión nominal es 5.0V.

<img src="./4_instrumentos.png" align="center" width="500" /> 


* 1_Dibuje el circuito correspondiente y describa el proceso para realizar la medición.
* 2_Seleccione el instrumento que utilizará en la medición justificando la elección.
* 3_Determinar todos los elementos necesarios para realizar el ensayo.
* 4_Si luego de tomar 10 muestras con y sin carga, se obtuvo que V1 = 0,0mV  σ1=0.1mV y V2 = 10,8mV σ2=1.1mV, hallar el resultado de la medición.



#### Punto 1

<img src="./circuito4.png" align="center" width="500" /> 

#### Punto 2

* Con la llave abierta, ajustar VREF (R_2 y R4) hasta lograr que la tensión indicada sea cero (Setear el voltimetro en la escala de 200mv).
* Cerrar la llave y tomar la medición.
* Abrir la llave nuevamente y verificar que la indicación vuelva a cero, de lo contrario se descarta la medición

#### Punto 3

Multímetro A:

Resolución: $\frac{200mV}{20000}=0,01mV$

$∆𝑉 =\frac{𝑒𝑟 \%}{100}𝑉 + 𝑛·𝑅𝑒𝑠𝑉𝑜𝑙 =\frac{2}{100}𝑉 + 3·0, 01𝑚𝑉 = 1, 03𝑚𝑉$


Multímetro B:

Resolución: $\frac{200mV}{2000}=0,1mV$

$∆𝑉 =\frac{𝑒𝑟 \%}{100}𝑉 + 𝑛·𝑅𝑒𝑠𝑉𝑜𝑙 =\frac{0,5}{100}𝑉 + 1·0, 01𝑚𝑉 = 0,35𝑚𝑉$


Multímetro C:

Resolución: $\frac{200mV}{2000}=0,1mV$

$∆𝑉 =\frac{𝑒𝑟 \%}{100}𝑉 + 𝑛·𝑅𝑒𝑠𝑉𝑜𝑙 =\frac{0,15}{100}𝑉 + 2·0, 1𝑚𝑉 = 0,275𝑚𝑉$


Cuentas usando V=50mV. 

Elijo el multímetro C porque el error % es lo que más pesa

#### Punto 4

VR:9V

Adopto para que no sea muy grande y descargue la bateria ni pequeña para que el ruido afecte. 

$Ir=100\mu A$

$\Delta V_s=50mV$

$V_{R2}>2\Delta V_s=100mV$

$V_{R2}>\frac{V_{R2}}{Ir}=\frac{100mv}{0,1ma}=2K \Omega$

Como la resolución del voltimetro C es de 0,1mv

$\frac{0,1\frac{mv}{Paso}}{0,1mA}=1\frac{\Omega}{paso}$

$\frac{2K\Omega}{1\frac{\Omega}{paso}}=2000 Pasos$

$\frac{2000 pasos \cdot 3º}{360º}=16,6$ Vueltas adopto un potenciometro de 25 vueltas.

$R_1=\frac{9V-(5V+50mv)}{0,1mA}=\frac{3,95V}{0,1mA}=39,5K\Omega => 39K\Omega$

Proponiendo el ajusto gureso en 1V

$R_3=\frac{V_{R3}}{Ir}=\frac{V_R-V_{R1}-V_{R2}-V_{R4}}{Ir}=\frac{9V-3,95V-100mv-1V}{0,1mA}=\frac{3,95V}{0,1mA}=39,5K\Omega=> 39K\Omega$

$R_{4}=\frac{V_{R4}}{Ir}=\frac{1V}{0,1mA}=10K\Omega$

$RL=\frac{5V}{1,5A}=3,3\Omega$

$P_L=5V\cdot1,5A=7,5W$


### TIPO A

In [11]:
N=10
V1m=0
SV1=0.1
uaV1=SV1/np.sqrt(N)
print('ua(V)=',uaV1,'mV') 

V2m=10.8
SV2=1.1
uaV2=SV2/np.sqrt(N)
print('ua(V)=',uaV2,'mV') 

ua(V)= 0.03162277660168379 mV
ua(V)= 0.3478505426185217 mV


### TIPO B

In [12]:
DELTAV1=(2*0.1)
ubV1=(DELTAV1/np.sqrt(3))

DELTAV2=((0.15/100)+(2/108))*V2m
ubV2=(DELTAV2/np.sqrt(3))
print('ub(V1)=',ubV1,'mV') 
print('ub(V2)=',ubV2,'mV')

ub(V1)= 0.11547005383792516 mV
ub(V2)= 0.12482312819879711 mV


### Combinada

In [13]:
ucV1=np.sqrt(ubV1**2+uaV1**2)
ucV2=np.sqrt(ubV2**2+uaV2**2)
DeltaV=V2m-V1m
uDeltaV=np.sqrt(ucV1**2+ucV2**2)
UDeltaV=2*uDeltaV
print('ucr(V1)=',ucV1,'mv')
print('ucr(V2)=',ucV2,'mv')
print('U(DeltaV)=',UDeltaV,'mV')


ucr(V1)= 0.11972189997378647 mv
ucr(V2)= 0.3695684149563289 mv
U(DeltaV)= 0.7769534005760362 mV


In [14]:
print(f"DeltaV=({DeltaV:.2f}±{UDeltaV.round(2):.2f})mV")
print("K=2 con el 95,45% de nivel de confianza")


DeltaV=(10.80±0.78)mV
K=2 con el 95,45% de nivel de confianza


La Regulación es buena menor a 50mV

### Ejercicio 5

Se desea calibrar la referencia de tensión de 1 V del laboratorio de Medidas Electrónicas I (calibrador Pioneer) a partir de un banco de pilas patrón de 10V.
 
Datos:
·         Patrón primario: Tensión de salida: 10V Exactitud de salida: 0.001% accuracy
·         Divisor Kelvin & Varley: Resolución : 1ppm Exactitud: 0.1 ppm
·         Calibrador Pioneer 450A: Salida de 1V ±0.05% accuracy
·         Voltímetro: Fluke 189


<img src="./ej5.png" align="center" width="500" /> 

* a) En base a las mediciones, informe el resultado final de la calibración y verifique el cumplimiento o no de las especificaciones.
* b) Explique el concepto de trazabilidad.
* c) Se realizó la medición directa de la salida del Pioneer 450A con el Fluke 189 arrojando los siguientes valores:

<img src="./ej52.png" align="center" width="500" /> 

En base a ellos defina cuál es el valor de la incertidumbre en la calibración del Pioneer 450A y compárelo con el calculado en el punto (a); extraiga conclusiones.


Circuito de medición:

<img src="./ej5circuito.png" align="center" width="500" /> 

Del circuito vemos que:

$V_i=V_p \cdot K - V_x$

$V_x=V_p \cdot K- V_i$

### TIPO A

In [17]:
Vpm=10000
Km=0.1
Vim=30
Vxm=Vpm*Km-Vim
Vxm

uaVi=0.001
uaK=0
uaVp=0

### TIPO B

In [20]:
DeltaVp=(0.001/100)*Vpm
ubVp=(DeltaVp/np.sqrt(3))

DeltaK=((0.1/(10**6)))
ubK=(DeltaK/np.sqrt(3))

DeltaVi=((0.1/100)+(20/30000))*Vim
ubVi=(DeltaVi/np.sqrt(3))
print('ub(Vp)=',ubVp,'mV') 
print('ub(K)=',ubK,'-')
print('ub(Vi)=',ubVi,'mV') 

ub(Vp)= 0.05773502691896258 mV
ub(K)= 5.7735026918962584e-08 -
ub(Vi)= 0.028867513459481287 mV


### Combinada

In [22]:
ucVp=np.sqrt(ubVp**2+uaVp**2)
ucK=np.sqrt(ubK**2+uaK**2)
ucVi=np.sqrt(ubVi**2+uaVi**2)
Vx=Vpm*Km-Vim
print('Vx=',Vx,'mV')


Vx= 970.0 mV


$$Vx=Vp \cdot K - V_i$$

$\frac{\partial V_x}{\partial V_p}=K$

$\frac{\partial V_x}{\partial K}=V_p$

$\frac{\partial V_x}{\partial V_i}=-1$

$\mu(V_x)=\sqrt{ \mu(V_p)^2\cdot\frac{\partial V_x}{\partial V_p}|_{K,V_i}+\mu(K)^2\cdot \frac{\partial V_x}{\partial K}|_{V_p,V_i}+\mu(V_i)^2 \cdot \frac{\partial V_x}{\partial V_i}|_{V_p,K}}$

In [23]:
uDeltaVx=np.sqrt((Km**2)*(ucVp**2)+(Vpm**2)*(ucK**2)+((-1)**2)*(ucVi**2))
UDeltaVx=2*uDeltaVx

print('U(DeltaV)=',UDeltaVx,'mV')

U(DeltaV)= 0.05892367945062493 mV


In [24]:
print(f"Vx=({Vx:.3f}±{UDeltaVx.round(3):.3f})mV")
print("K=2 con el 95,45% de nivel de confianza")

Vx=(970.000±0.059)mV
K=2 con el 95,45% de nivel de confianza


De la especificación teniamos por defecto:

1V-0.05%=0.9995V=999.5mV

De lo medido

Vx=(970.000-0.059)mv=0.969,941mV

No cumple la especificación

### Trazabilidad:

La trazabilidad es la cadena que se establece entre los instrumentos de medición desde el patrón hacia los diferentes elementos de medición. De esta forma podemos asegurar que cuando medimos con un instrumento que fue calibrado con un elemento superior en la cadena (y así dicho elemento con el superior, etc), nuestra medición guarda un correlato de medición con el patron primario. Decimos entonces que nuestra medición es trazable con el patron en cuestión.

Podemos ver el concepto de trazabilidad representado en un formato de piramide.


<img src="./piramide.jpg" align="center" width="500" /> 


### Medicion directa

### Tipo A

In [25]:
V=1001.5
uaV=0.001
print('ua(V)=',uaV,'mV') 

ua(V)= 0.001 mV


### Tipo B

In [26]:
DeltaV=((0.025/100)+(5/10015))*V
ubV=(DeltaV/np.sqrt(3))
print('ub(V)=',ubV,'mV') 

ub(V)= 0.43322920824316546 mV


### Combinada

In [27]:
ucV=np.sqrt(ubV**2+uaV**2)
UV=2*ucV
print(f"V=({V:.2f}±{UV.round(2):.2f})mV")
print("K=2 con el 95,45% de nivel de confianza")

V=(1001.50±0.87)mV
K=2 con el 95,45% de nivel de confianza


De la especificación:

1V-0.05%=0.95V=999.5mV
1V+0.05%=1.05V=1000.5 mv

Ahora de las nuevas medidas:

(1001.50-0.87)mv=1000.63 mv
(1001.50+0.87)mv=1002.37 mv

Seguimos estando fuera de las especificaciones, pero en la medición directa tenemos una incertidumbre mayor que en la medición de calibración realizada utilizando el metodo diferencial con el patron de 10V y el divisor. En este segundo caso logramos trasladar la incertidumbre (combinada de los elementos en el circuito de medición) y al estos tener menor incertidumbre obtuvimos una calibración mejor.

### Ejercicio 6

Se midieron por comparación una resistencia patrón y una incógnita con un voltímetro digital
obteniendo los siguientes resultados:

Vp = 0,9863 V y un desvió estándar experimental de 0.05 mV

Vx = 1,7588 V y un desvió estándar experimental de 0.05 mV

Sabiendo que: Rp= 10MΩ ±0,05% 0,1W Rx≈ 20MΩ 1⁄4W

Y el voltímetro tiene las siguientes características: Dígitos: 4 3⁄4 Rv= 10Mohm

Error: 40mV ±(0,5%+2d) ; 400mV ±(0,5%+2d) ; 4V ±(0,5%+3d) ; 40V ±(0,5%+3d) ; 400V ±(0,5%+3d)

* a) Dibuje el circuito correspondiente
* b) Deduzca las ecuaciones que rigen la ecuación planteada
* c) Determine el resultado de la medición
* d) Critique la medición indicando las modificaciones que realizaría para mejorarla indicando cual es el impacto de los cambios sugeridos.


#### Punto A

<img src="./ej6.png" align="center" width="300" /> 

#### Punto B

$V_p'=V_s\frac{Rp//Rv}{(R_p//R_V)+R_s+R_x}$

$V_p'=V_s\frac{\frac{R_p\cdot R_v}{R_p+R_v}}{\frac{R_p\cdot R_v}{R_p+R_v}+R_s+R_x}$

$V_p'=V_s\frac{R_p\cdot R_V}{R_P\cdot R_V+(R_P+R_V)\cdot(R_S+R_x)}$


$V_x'=V_s\frac{Rx//Rv}{(R_x//R_V)+R_s+R_P}$

$V_x'=V_s\frac{\frac{R_x\cdot R_v}{R_x+R_v}}{\frac{R_p\cdot R_v}{R_p+R_v}+R_s+R_P}$

$V_x'=V_s\frac{R_x\cdot R_V}{R_x\cdot R_V+(R_x+R_V)\cdot(R_S+R_P)}$

$V_p'=0,9863V$

$V_x'=1,7588V$

$\frac{V_p}{V_x}=\frac{V_s\frac{R_p\cdot R_V}{R_P\cdot R_V+(R_P+R_V)\cdot(R_S+R_x)}}{V_s\frac{R_x\cdot R_V}{R_x\cdot R_V+(R_x+R_V)\cdot(R_S+R_P)}}$

$\frac{V_p}{V_x}=\frac{\frac{R_p}{R_P\cdot R_V+(R_P+R_V)\cdot(R_S+R_x)}}{\frac{R_x}{R_x\cdot R_V+(R_x+R_V)\cdot(R_S+R_P)}}$

$\frac{V_p}{V_x}=\frac{R_x}{R_P}\cdot \frac{R_P\cdot R_V+(R_P+R_V)\cdot(R_S+R_x)}{R_x\cdot R_V+(R_x+R_V)\cdot(R_S+R_P)}$

$R_x=\frac{V_x}{V_p}\cdot R_p \cdot \frac{R_x\cdot R_V+(R_x+R_V)\cdot(R_S+R_P)}{R_P\cdot R_V+(R_P+R_V)\cdot(R_S+R_x)}$

$Rx_i=\frac{Vx}{Vp}\cdot Rp$

$Rx_i=\frac{1,7588V}{0,9863V}\cdot 10M\Omega=17,8323025449M\Omega$

$Kc=\frac{R_x\cdot R_V+(R_x+R_V)\cdot(R_S+R_P)}{R_P\cdot R_V+(R_P+R_V)\cdot(R_S+R_x)}$

$Rx=Rx_i Kc$

$e{M_r}=\frac{Rx_i-Rx}{Rx}$

$e{M_r}=\frac{Rp \frac{Vx}{Vp}-Rp \frac{Vx}{Vp} Kc}{Rp\frac{Vx}{Vp}Kc}$

$e{M_r}=\frac{1-Kc}{Kc}=\frac{1}{Kc}-1$

$e{M_r}=\frac{R_P\cdot R_V+(R_P+R_V)\cdot(R_S+R_x)}{R_x\cdot R_V+(R_x+R_V)\cdot(R_S+R_P)}-1$

No corregimos por error de metodo dado que en los casos prácticos $RS<10\Omega$ y $Rv>10M\Omega$ como nuestro caso.


#### Punto C

$\mu_r(Rx)=\sqrt{\mu_r(Rp)^2+\mu_r(Vx)^2+\mu_r(Vp)^2}$

$Ur(Rp)=0,05$
$\mu_r(Rp)=Ur(Rp)/2=0,025$


### Tipo A

In [106]:
Rx=17.8323025449
Vx=1.7588
Vp=0.9863
uarVx=0.05/(1000*Vx)
uarVp=0.05/(1000*Vp)
print('uar(Vx)=',uarVx) 
print('uar(Vp)=',uarVp) 


uar(Vx)= 2.8428473959517855e-05
uar(Vp)= 5.069451485349286e-05


### Tipo B

In [107]:
DELTAVx=((0.5/100)+(3/17588))
ubrVx=(DELTAVx/np.sqrt(3))
DELTAVp=((0.5/100)+(3/9863))
ubrVp=(DELTAVp/np.sqrt(3))
print('ubr(Vx)=',ubrVx) 
print('ubr(Vp)=',ubrVp)

ubr(Vx)= 0.0029852304685071963
ubr(Vp)= 0.003062362296730739


### Combinada

In [108]:
ucrVx=np.sqrt(ubrVx**2+uarVx**2)
ucrVp=np.sqrt(ubrVp**2+uarVp**2)
urRp=0
urRx=np.sqrt(ucrVx**2+ucrVp**2+(urRp)**2)
uRx=urRx*Rx
URx=2*uRx
print('U(Rx)=',URx,'MΩ')

U(Rx)= 0.15253882054635498 MΩ


In [109]:
print(f"U(Rx)=({Rx:.2f}±{URx.round(2):.2f})MΩ")
print("K=2 con el 95,45% de nivel de confianza")


U(Rx)=(17.83±0.15)MΩ
K=2 con el 95,45% de nivel de confianza


#### Punto D

En estos casos no se utilizó a fondo de escala el multimetro para que la incertidumbre de las mediciones tenga un impacto menor, para esto propongo aumentar la tensión de fuente a modo de medir a fondo de escala.

### Ejercicio 7

En un puente universal determine:
* a) El circuito correspondiente a la medición de un inductor de bajo factor de merito
* b) Describa el proceso de medición
* c) Si se opera con una fuente de 10KHz, ¿Cuáles serán los resultados a obtener?
* d) ¿Qué diferencia representa materializar un capacitor real en configuración paralelo o configuración serie?


#### Punto A

Utilizaria un puente de Maxwell:

<img src="./puentemaxwell.png" align="center" width="300" /> 

#### Punto B

Procedimiento para realizar la medición:


* 1_Seleccionar el tipo de puente (Maxwell).
* 2_Seleccionar el rango adecuado ( R2)
* 3_Colocar la sensibilidad del detector al mínimo.
* 4_Energizar el puente.
* 5_Ajustar el valor del componente (R1) hasta encontrar un mínimo local en el detector. De ser necesario aumentar la sensibilidad del detector.
* A_justar el valor del Q (R3) hasta encontrar un mínimo local en el detector. De ser necesario aumentar la sensibilidad del detector.
* 7_Repetir los pasos 5 y 6 hasta lograr el cero en el detector en la con la mayor sensibilidad posible.

#### Punto C

$Q_x=\frac{f_x}{f_0}\cdot Q_i=\frac{10KHz}{1KHz}\cdot Q_i=10Q_i$

$Q_x=\frac{\omega L_x}{R_x}=10 \cdot \omega \cdot R_1 \cdot  C_1$

$\frac{\omega L_x}{\frac{R_2 \cdot R_3}{R_1}}=10\cdot \omega \cdot R_1 \cdot \ C_1$

$L_x=10\cdot R_2 \cdot R_3 \cdot C_1 =10 \cdot L_x$

La inductancia se multipla por $\frac{f_x}{f_0}$ en esta caso 10


#### Punto D

$Z_s=R_s+\frac{1}{j\omega C_s}$

$Q_s=\frac{1}{\omega R_s C_s}$

$Zp=\frac{1}{\frac{1}{R_p}+j\omega C_p}$

$Q_p=\omega C_p R_p$

$$Z_s=Z_p$$

$R_s=\frac{1}{j\omega C_s}=\frac{1}{\frac{1}{R_p}+j\omega C_p}$

$(R_s+\frac{1}{j\omega C_s})\cdot(\frac{1}{R_p}+j\omega C_p)=1$

$\frac{R_s}{R_p}+j\omega C_p R_s+\frac{1}{j\omega C_s R_p}+\frac{j\omega C_p}{j\omega C_s}=1$

Igualando parte real y parte imaginaria:

De la parte imaginaria:

$\omega C_p R_s=\frac{1}{\omega C_s R_p}$

$\omega C_p R_p=\frac{1}{\omega C_s R_s}$

$$Q_s=Q_p$$

De la parte real:

$\frac{R_s}{R_p}+\frac{C_p}{C_s}=1$

$\frac{\frac{1}{Q\omega C_s}}{\frac{Q}{\omega C_p}}+\frac{C_p}{C_s}=1$

$\frac{C_p}{C_s}(\frac{1}{Q^2}+1)=1$

$$C_s=C_p(\frac{1}{Q^2}+1)$$

Conclusión: para pasar de un Sauty Paralelo a un Sauty Serie, el $C_x$ va a depender del Q. Si por ejemplo el Q = 1, $C_S = C_P$ y el capacitor no cambia. Si Q tiende a infinito, entonces $C_S=2C_P$
